integration-tests/coverage-remapping-inline-sourcemap/preprocessor.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const jestPreset = require('babel-preset-jest');
+const babelTransform = require('babel-core').transform;
+const babelUtil = require('babel-core').util;
+const babelIstanbulPlugin = require('babel-plugin-istanbul').default;
+
+const options = {
+  presets: ['env', jestPreset],
+  retainLines: true,
+  sourceMaps: 'inline',
+};
+
+module.exports = {
+  canInstrument: true,
```

Comment:
Should I call it `coverage-remapping-inline-sourcemap-instrumented`? Basically I wanna say
* I'm asserting on the coverage map that was generated
* This is returning `inline-sourcemap`
* This instruments

Reply:
Maybe `coverage-remapping-instrumented` makes more sense. Since returned `inline-sourcemap` is not affecting coverage

Reply:
Maybe pick the most important thing you're trying to assert on and name the test around that. My biggest concern is that 'coverage-remapping' in the name is misleading if the transformer is doing instrumentation for coverage, because instrumenting transformers don't hit the coverage remapping code paths.